espnet2/asr/espnet_model.py

```diff
@@ -182,7 +185,11 @@ def collect_feats(
         text: torch.Tensor,
         text_lengths: torch.Tensor,
     ) -> Dict[str, torch.Tensor]:
-        feats, feats_lengths = self._extract_feats(speech, speech_lengths)
+        if self.extract_feats_for_collect_stats:
```

Comment:
Can someone check this part?
I modified this because it is pain to run collect_feats for HuBERT / Wav2Vec2 frontend. On the other hand, if utt_mvn is used, then I feel it is not necessary to run this step, although I'm not sure.

Reply:
Do you need me to run a comparison experiment?

Reply:
Thanks! I've run some tests. If you could test it again, that would be very much helpful.
First it would not effect the cases when we run the "_extract_feats". Also, it works for this PR, using HuBERT / Wav2Vec2 frontend and setting `extract_feats_for_collect_stats=false`. I'm not sure if there are some other things I didn't think of.

Reply:
Does the unit test go through this part?
If so, it would be OK.
If not, maybe we can make a unit test?

Reply:
Sounds good. I added a test in `test_integration`.

Reply:
Year, then, this should be OK.